### DISCUSSION TITLE
Uptime

### DISCUSSION THREAD

**Discussion by koaning** [plausible/analytics#962] [Anything Else]
This is a strange feature request perhaps, but I was wondering ... why not add an uptime feature? It feels like an extra feature that some folks would be willing to pay for. I'm showing my stats publicly anyway, I'd love for uptime to also be listed.

**Comment by metmarkosaric** [comment]
thanks @koaning! could be cool to explore the performance / speed / uptime metrics in the future

**Comment by KasparEtter** [comment]
I have a slightly different take on this. Besides having traffic spike notifications, it would be nice to have “traffic silence notifications”: Configure after how many hours of no hits to Plausible you would like to get an email. Given that this would require you to monitor the traffic over time, it might not be easy to implement, though.
